lib/ansible/modules/windows/win_command.py

```diff
@@ -60,17 +60,16 @@
 '''
 
 EXAMPLES = r'''
-# Example from Ansible Playbooks.
-- win_command: whoami
+- name: Save the result of 'whoami' in 'whoami_out'
+  win_command: whoami
   register: whoami_out
 
-# Run the command only if the specified file does not exist.
-- win_command: wbadmin -backupTarget:C:\backup\ creates=C:\backup\
+- name: Create a backup only if the file does not exist
+  win_command: wbadmin -backupTarget:C:\backup\
+    creates: C:\backup\
 
-# You can also use the 'args' form to provide the options. This command
-# will change the working directory to C:\somedir\\ and will only run when
-# C:\backup\ doesn't exist.
-- win_command: wbadmin -backupTarget:C:\backup\ creates=C:\backup\
+- name: Move to C:\somedir\ and create a backup only if the file does not exist leveraging the args to change directory and declare what will be created
```

Comment:
This line is too long (for a `name:` key). My solution for things like this is to put only the performed action in the `name:` key, and move most of the explanation in a comment right above. (So split the current comment into a shorter comment, and a `name:` statement).

To be honest, I don't think we need most of the comment as we modified the previous example in exactly the same way as this example. I think we can merge both examples, and get rid of the creates= example (that's legacy notation that we don't want to promote too much, certainly not as the first example in the list).